EIPS/eip-template.md

```diff
@@ -0,0 +1,86 @@
+---
```

Comment:
```suggestion
---
eip: 6799
```

This draft PR 6800 was created two minutes after #6799.
To avoid any perception that important core EIPs get special treatment and to discourage number sniping, gaming or even a number lunge as a trial EIP editor I am assigning this as EIP6799.

EIP editors are free to override this.

Examples of number renaming discussions:
https://github.com/ethereum/EIPs/pull/5000#discussion_r850655921
https://github.com/ethereum/EIPs/pull/4888#pullrequestreview-905617398

Reply:
```suggestion
---
eip: 6800
```
Assigned by @lightclient in comment: https://github.com/ethereum/EIPs/pull/6800/commits/43bb39436da52dbc5aaa46786da404e929224f86#r1152654153